webapp/components/TooltipNew/TooltipNew.js

```diff
@@ -0,0 +1,43 @@
+import './TooltipNew.scss'
+
+import React, { useCallback, useState } from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+import { Tooltip } from '@mui/material'
+
+import { useI18n } from '@webapp/store/system'
+
+import Markdown from '../markdown'
+
+export const TooltipNew = (props) => {
+  const { children, className, titleInMarkdown, title: titleProp, renderTitle } = props
```

Comment:
ok

Reply:
I just wanted to avoid using a verb in the parameter name, it looks like a function to me in that way, but you are right, `titleInMarkdown` seems like "the title not in text but in markdown"